plugins/Eq/EqControlsDialog.cpp

```diff
@@ -72,17 +72,17 @@ EqControlsDialog::EqControlsDialog( EqControls *controls ) :
 	setBand( 6, &controls->m_highShelfActiveModel, &controls->m_highShelfFreqModel, &controls->m_highShelfResModel, &controls->m_highShelfGainModel, QColor(255 ,255, 255), tr( "High-shelf" ), &controls->m_highShelfPeakL, &controls->m_highShelfPeakR,0,0,0,0,0,0 );
 	setBand( 7, &controls->m_lpActiveModel, &controls->m_lpFreqModel, &controls->m_lpResModel, 0, QColor(255 ,255, 255), tr( "LP" ) ,0,0,0,0,0, &controls->m_lp12Model, &controls->m_lp24Model, &controls->m_lp48Model);
 
-	auto faderBg = new QPixmap(PLUGIN_NAME::getIconPixmap("faderback"));
-	auto faderLeds = new QPixmap(PLUGIN_NAME::getIconPixmap("faderleds"));
-	auto faderKnob = new QPixmap(PLUGIN_NAME::getIconPixmap("faderknob"));
+	auto faderBg = QPixmap(PLUGIN_NAME::getIconPixmap("faderback"));
+	auto faderLeds = QPixmap(PLUGIN_NAME::getIconPixmap("faderleds"));
+	auto faderKnob = QPixmap(PLUGIN_NAME::getIconPixmap("faderknob"));
```

Comment:
`Fader` maintains pointers to the pixmaps internally, so this will result in a use-after-free once these go out of scope.